tutorials/language-decl/index.uk.html

```diff
@@ -1,9 +1,9 @@
 <!DOCTYPE html>
 <html lang="uk">
 <head>
-<meta charset="utf-8" />
+<meta charset="utf-8">
 <title>Робота з мовою в HTML</title>
-<meta name="description" content="Посібник W3C i18n: як призначити і використовувати мову в XHTML та HTML файлах" />
+<meta name="description" content="Посібник W3C i18n: як призначити і використовувати мову в XHTML та HTML файлах">
 <script type="application/javascript">
```

Comment:
We should remove instances of type="application/javascript" and type="text/javascript"

Reply:
Fixed.